### PR TITLE
Add tabnews extension

### PR DESCRIPTION
**Pull request by juniorodilton**
## Description
This extension allows you to see the content from [Tabnews](https://www.tabnews.com.br/)

## Screencast
screenshot

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/tabnews/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          icon={getIcon(index + 1)}
+          accessories={[
+            { icon: Icon.Bubble, text: item.children_deep_count.toString() },
+            {
+              icon: Icon.StarCircle,
+              text: item.tabcoins.toString(),
+            },
+          ]}
+          actions={
+            <ActionPanel title={item.title}>
```
Comment (pernielsentikaer) [inline extensions/tabnews/src/index.tsx:41]:
```suggestion
            <ActionPanel title="Options">
```

File: extensions/tabnews/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+              <ActionPanel.Section>
+                {<Action.OpenInBrowser url={getContetUrlFromPostResponse(item)} />}
+              </ActionPanel.Section>
+              <ActionPanel.Section>
+                {<Action.CopyToClipboard content={getContetUrlFromPostResponse(item)} />}
+              </ActionPanel.Section>
+            </ActionPanel>
```
Comment (pernielsentikaer) [inline extensions/tabnews/src/index.tsx:48]:
```suggestion
              <ActionPanel.Section>
                {<Action.OpenInBrowser url={getContetUrlFromPostResponse(item)} />}
                {<Action.CopyToClipboard content={getContetUrlFromPostResponse(item)} />}
              </ActionPanel.Section>
            </ActionPanel>
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Maybe add a preference to let the user select more than 10 items to fetch?

- Add [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/tabnews/src/index.tsx (hunk 1 above)
Comment (juniorodilton) [reply extensions/tabnews/src/index.tsx]: Thanks, this make really more sense! I'll apply your suggestion!

File: extensions/tabnews/src/index.tsx (hunk 2 above)
Comment (juniorodilton) [reply extensions/tabnews/src/index.tsx]: Sure, this is more clean! I'll apply your suggestion!

**Comment by juniorodilton** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> * Maybe add a preference to let the user select more than 10 items to fetch?
> * Add [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots)
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Im already working to use `preferencesApi` to setup the pageLimit and user account, I'd like to do this in next update of this extension :)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**